docs/python-integration.md

```diff
@@ -0,0 +1,299 @@
+# SQLAlchemy Integration
+
+pgai provides SQLAlchemy integration for managing and querying vector embeddings in your database through a simple declarative interface.
+
+## VectorizerField
+
+### Basic Setup
+
+```python
+from sqlalchemy import Column, Integer, Text
+from sqlalchemy.orm import DeclarativeBase
+from pgai.sqlalchemy import VectorizerField
+from pgai.configuration import EmbeddingConfig
+
+class BlogPost(DeclarativeBase):
+    __tablename__ = "blog_posts"
+    
+    id = Column(Integer, primary_key=True)
+    title = Column(Text, nullable=False)
+    content = Column(Text, nullable=False)
+    
+    content_embeddings = VectorizerField(
```

Comment:
I think this is a great interface. One nit `VectorizerField` or `Vectorizer`? In my mind this isn't really a field...